main.py

```diff
@@ -0,0 +1,69 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+import crud
+import models
+import schemas
+from database import SessionLocal
+
+app = FastAPI()
```

Comment:
you must create all models on start app

Reply:
"you must create all models on start app".   What do you mean?

Reply:
https://fastapi.tiangolo.com/tutorial/sql-databases/#main-fastapi-app

Reply:
@AlenOl not fixed